peripherals/gpio/gpio_wl.yaml

```diff
@@ -1,22 +1,17 @@
-# Some GPIO appear to have BRR registers
+# This GPIO is used on the STM32WL families.
+
+_include:
+  - v2/common.yaml
 
 "GPIO[AB]":
   BRR:
-    "BR[0123456789]":
-      _write:
-        NoAction: [0, "No action on the corresponding ODx bit"]
-        Reset: [1, "Reset the ODx bit"]
-    "BR1[012345]":
+    "BR[0123456789],BR1[012345]":
```

Comment:
```suggestion
    "BR[0-9],BR1[0-5]":
```

Reply:
I've done the requested modifications in [a1b7ff3](https://github.com/stm32-rs/stm32-rs/commit/a1b7ff3f444898f1ddad9348273cb7b9fc44b03f).